README.md

```diff
@@ -15,30 +15,44 @@ It offers a convenient way to keep your logs when you are running unit tests.
 ```
 
 ## How to use
+### Global logger
 
-By default, it is initialized to wrap `logrus` package. You can override `log.Factory` to use the logger library you want.
+By default, it is initialized to wrap `logrus` package. You can use `log.SetFactory` to use the logger library you want.
```

Comment:
The feature is interesting but the fact that you replace log.Factory by the function log.SetFactory is a breaking change.
Please keep log.Factory as it

Reply:
done 👍🏻